cli/internal/cache/cache.go

```diff
@@ -271,6 +277,21 @@ func (mplex *cacheMultiplexer) Fetch(target string, key string, files []string)
 	return false, files, 0, nil
 }
 
+func (mplex *cacheMultiplexer) Exists(target string) (CacheState, error) {
+	syncCacheState := CacheState{}
+	for _, cache := range mplex.caches {
+		cacheState, _ := cache.Exists(target)
+		if cacheState.Local == true {
+			syncCacheState.Local = cacheState.Local
+		}
+		if cacheState.Remote == true {
+			syncCacheState.Remote = cacheState.Remote
+		}
+	}
```

Comment:
I don't love this - but couldn't think of a better way at the moment.

Reply:
This is fine. If you wanted to tighten it up a little you could probably do something like:
```go
syncCacheState.Local = syncCacheState.Local || cacheState.Local
syncCacheState.Remote = syncCacheState.Remote || cacheState.Remote
```